book.tex

```diff
@@ -0,0 +1,40 @@
+\documentclass[a4paper]{book}
+
+\usepackage[ngerman]{babel}
+\usepackage[T1]{fontenc}
+\usepackage{fontspec}
+\usepackage{titlesec}
+\usepackage[a4paper]{geometry}
+\usepackage[most]{tcolorbox}
+\titleformat{\chapter}{\Large\bfseries}{}{0em}{}
+
+\def \thesender {}
+\def \thecontent {}
+\def \thedatetime {}
+
+\newcommand{\fbmessage}[1]{%
```

Comment:
Use 3 arguments and omit the `\thesender` and `thedatetime` stuff?
`\fbmessage{$sender}{$date}{$escapedContent}` is much slimmer

Reply:
Found it more readable and extensible this way.